libs/langchain-community/src/vectorstores/azure_aisearch.ts

```diff
@@ -663,7 +663,7 @@ export class AzureAISearchVectorStore extends VectorStore {
         defaultConfigurationName: "semantic-search-config",
         configurations: [
           {
-            name: "semantic-search-config",
+            name: this.options.semanticConfigurationName ?? "semantic-search-config",
```

Comment:
This one should not change, as it's meant to be the default one: if the name changes here, the default schema will break.
This parameter was only meant (for now) to be used alongside a custom schema, not to override the default schema

Reply:
Bump on this @keiohtani!

I will apply this in a few days to move on on this.